nilearn/experimental/plotting/_surface_plotting.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+from nilearn import plotting as old_plotting
+
+
+def plot_surf(img, hemi=None, mesh=None, views=["lateral"], **kwargs):
+    """Plot a SurfaceImage.
+
+    TODO: docstring.
+    """
+    if mesh is None:
+        mesh = img.mesh
+
+    if hemi is None:
+        hemi = list(img.data.keys())
+    if isinstance(hemi, str):
+        hemi = [hemi]
+
+    fig, axes = plt.subplots(
+        len(views),
+        len(hemi),
+        subplot_kw={"projection": "3d"},
+        figsize=(4 * len(hemi), 4),
+    )
+    axes = np.atleast_2d(axes)
+
+    title = ""
+    if "title" in kwargs:
+        title = f"{kwargs.pop('title')} - "
+
+    for view, ax_row in zip(views, axes):
+        for ax, mesh_part in zip(ax_row, hemi):
```

Comment:
I don't we want our plotting functions to be that complex.
Should let users decide how they ant to arrange their subplots.